lib/dal/src/attribute/value/dependent_value_graph.rs

```diff
@@ -364,7 +410,20 @@ impl DependentValueGraph {
     pub fn independent_values(&self) -> Vec<AttributeValueId> {
         self.graph
             .externals(Outgoing)
-            .filter_map(|node_idx| self.graph.node_weight(node_idx).copied())
+            .filter_map(|node_idx| {
+                if let Some(attribute_value_id) = self.graph.node_weight(node_idx).copied() {
+                    if self
+                        .exceptions_to_independent_values
+                        .contains(&attribute_value_id)
+                    {
+                        None
+                    } else {
+                        Some(attribute_value_id)
+                    }
+                } else {
+                    None
+                }
+            })
```

Comment:
Exceptions in action. As of now, these are only the direct dependents of the secret. They are technically not "independent" despite the graph relationship.

Reply:
This is outdated